README.md

```diff
@@ -49,12 +49,9 @@ To get yourself set up with all the project's dependencies:
 git clone https://github.com/artsy/metaphysics
 cd metaphysics
 
-# Install node modules
-yarn install
-# Installed system tools (like cache servers)
-brew bundle
-# Get set up with a default env
-cp .env.oss .env
+# Run the setup script
+
+source ./scripts/setup.sh
 ```
 
 For Artsy staff wanting to expand on your `.env`, you can use `hokusai staging env get` to see staging's env vars. Or better yet, use [copy_env][] to match
```

Comment:
I think we can drop this section, since it overlaps/conflicts with what's done by the setup script.